### PR TITLE
[#381]imporvement(server): Check JAVA_HOME and HADOOP_HOME in start-shuffle-server.sh

### PR DESCRIPTION
**Pull request by cchung100m**
### What changes were proposed in this pull request?

- Comment the variable `JAVA_HOME` and `HADOOP_HOME` assignment in `bin/rss-env.sh`
- Keep the variable `JAVA_HOME` and `HADOOP_HOME` be checked in `bin/utils.sh`

### Why are the changes needed?

Fix: #381 

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

current UT

### CODE REVIEW

**Comment by xianjingfeng** [comment]
This tips maybe helpful for you. @cchung100m 
In fact, we only need to comment the following two lines of code
https://github.com/apache/incubator-uniffle/blob/897613b8022ea73d5664904bbf43fc27f93505cc/bin/rss-env.sh#L24-L25

And all scripts have used the following method.
https://github.com/apache/incubator-uniffle/blob/897613b8022ea73d5664904bbf43fc27f93505cc/bin/utils.sh#L166-L173

File: build_distribution.sh [hunk 1]
```diff
@@ -97,7 +97,7 @@ cd $RSS_HOME
 
 if [ -z "$JAVA_HOME" ]; then
   echo "Error: JAVA_HOME is not set, cannot proceed."
-  exit -1
```
Comment (xianjingfeng) [inline build_distribution.sh:100]: Why do we need to change this line?

Comment (cchung100m) [reply build_distribution.sh:100]: Hi @xianjingfeng 

there is a warning  reported by my IDE,  and I also see the return value is `1` in other file(e.g., `bin/start-shuffle-server.sh `)

`build_distribution.sh`
image


`bin/start-shuffle-server.sh`
image

Comment (xianjingfeng) [reply build_distribution.sh:100]: > there is a warning reported by my IDE, and I also see the return value is `1` in other file(e.g., `bin/start-shuffle-server.sh `)

We can fix it in next pr.

Comment (cchung100m) [reply build_distribution.sh]: Sure, thanks for friendly reminder and I updated the part you mentioned.

**Review by xianjingfeng — APPROVED** [review]
LGTM

**Pull request merged**